14/umbraco-commerce/release-notes/README.md

```diff
@@ -23,7 +23,7 @@ This section contains the release notes for Umbraco Commerce 14 including all ch
 
 #### 14.1.0 (25th September 2024)
 
-* Added [Sales Tax Provider](../../key-concepts/sales-taxt-providers.md) support.
+* Added [Sales Tax Provider](../key-concepts/sales-taxt-providers.md) support.
```

Comment:
This link doesn't work in the preview 🤔

Reply:
When I preview it opens in Github for me 😳 Not sure what's going on here. 🤷‍♀️